app/repositories/TagRepository.scala

```diff
@@ -108,6 +110,17 @@ case class TagSearchCriteria(
 
   private def typeFilter(types: List[String])(tags: List[Tag]) = tags.filter { t => types.contains(t.`type`.toLowerCase) }
 
+  private def hasFieldsFilter(fields: List[String])(tags: List[Tag]) = tags.filter { t =>
+    fields.forall {
+        case "contributorinformation.bylineimage" => t.contributorInformation.flatMap(_.bylineImage).isDefined
+        case "contributorinformation.largebylineimage" => t.contributorInformation.flatMap(_.largeBylineImage).isDefined
+        case unsupported => {
+          Logger.warn(s"Attempted to check if tag has field '$unsupported' which is not supported by the hasFieldsFilter")
```

Comment:
Could we communicate this failure back to the client?

Reply:
Was previously just defaulted to true if it was unsupported... might just go back to that.